data/help_templates/find-src-in-translated-help-missing-in-reference.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# print src files referenced in translated help pages, but not in the reference (en)
+
+DATA_DIR="$PWD/.."
+CHECK_FILE="help_templates/orig/index.tml"
+if [ -f "$1"/"$CHECK_FILE" ] ; then
+	DATA_DIR=$1
+elif [ -n "$1" ] || [ ! -f "$DATA_DIR"/"$CHECK_FILE" ] ; then
+	echo cannot find the data dir, set it with something like:
+	echo $0 ~/lincity-ng/data
+	exit
+fi
+cd $DATA_DIR
+
+for H in help help_templates
+do
+	echo
+	echo === directory: $H ===
+	cd $H
+	CONTENT_FILES=`find -type f -iname "*.xml" | xargs grep src | cut -d'"' -f 2 | sort | uniq`
+
+	if [ $H = "help" ]; then
+		REF="en"
+	elif [ $H = "help_templates" ]; then
+		REF="orig"
+	else
+		echo error
+		exit
+	fi
+	echo reference directory: $REF
+
+	cd $REF
+	XML_FILES=`find -type f -iname "*.xml" | sort`
+
+	for F in $CONTENT_FILES
+	do
+		_F=`echo $F | rev | cut -d'/' -f 1 | rev`
+		grep [/\"]$_F $XML_FILES > /dev/null
```

Comment:
Use `grep -m 1` for performance. Also consider adding `--` to explicitly separate the files to search from the regex.